cli/src/create/utils.rs

```diff
@@ -0,0 +1,205 @@
+use std::fs;
+use std::fs::File;
+use std::io::Read;
+use std::io::Write;
+use std::path::Path;
+use std::path::PathBuf;
+
+use anyhow::{Error, Result};
+use include_dir::include_dir;
+use include_dir::Dir;
+use indicatif::ProgressBar;
+use indicatif::ProgressStyle;
+use reqwest::blocking::Client;
+use zip::ZipArchive;
+
+use super::APP;
+
```

Comment:
All the functions here were copied from `create.rs` without any changes except `check_bindings`